controller/lib/commands/command_handlers.h

```diff
@@ -2,7 +2,7 @@
 #define __COMMAND_HANDLERS_H
 #include "network_protocol.pb.h"
 
-void command_handler(Command);
-void ack_handler(GuiAck);
+extern void command_handler(Command);
+extern void gui_ack_handler(GuiAck);
```

Comment:
"extern" isn't necessary here in C++ (nor in C, I think).

Reply:
it isn't, but it documents that this is intended to be provided

Reply:
Hm, for whatever reason I don't have that same cultural understanding.  In fact I'm not even sure what you mean by "overridden" -- does this have to do with mocking (which I was hoping we could move into hal)?

Could we leave a comment?